### PR TITLE
[Test][Model Zoo] enable model zoo test

### PR DESCRIPTION
**Pull request by xuhongyao**

### CODE REVIEW

File: lib/parser/onnx/onnx_parser.cc [hunk 1]
```diff
@@ -333,10 +333,6 @@ Tensor<T> ONNXParser::ProcessTensor(const onnx::TensorProto& tensor_proto) {
     LOG(ERROR) << "Unsupported external data storage.";
   }
 
-  if (shape.empty() && !v.empty()) {
```
Comment (weimingzha0) [inline lib/parser/onnx/onnx_parser.cc:336]: If the v.size() > 1, should we set the shape?

File: models/vision/classification/alexnet/run_alexnet.sh [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# RUN: %s
```
Comment (weimingzha0) [inline models/vision/classification/alexnet/run_alexnet.sh:2]: can we also check the output?

File: models/lit.cfg.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+config.excludes = ['run.all.sh', 'get_images.sh']
+
+config.environment['SRC_DIR'] = config.halo_src_dir
+config.environment['BUILD_DIR'] = config.halo_build_dir
+config.environment['ODLA_INC'] = os.path.sep.join(
+    (config.halo_src_dir, 'ODLA', 'include'))
+config.environment['ODLA_LIB'] = os.path.sep.join(
+    (config.halo_build_dir, 'lib'))
+config.environment['TEST_TEMP_DIR'] = '/tmp'
+config.environment['TEST_WITH_GPU'] = '1'
```
Comment (weimingzha0) [inline models/lit.cfg.py:21]: check if based on the availability of libodla_tensorrt?

File: models/vision/classification/alexnet/run_alexnet.sh (hunk 2 above)
Comment (weimingzha0) [reply models/vision/classification/alexnet/run_alexnet.sh:2]: or we can do it in a follow-up patch

File: lib/parser/onnx/onnx_parser.cc (hunk 1 above)
Comment (xuhongyao) [reply lib/parser/onnx/onnx_parser.cc:336]: yes, you are right, we should check shape.empty && v.size() > 1

File: models/lit.cfg.py (hunk 3 above)
Comment (xuhongyao) [reply models/lit.cfg.py]: yes, I have used cmake option ""@cudart@"" to check it

**Review by weimingzha0 — APPROVED** [review]

**Pull request merged**